src/features/areaAssignments/components/AreaSelect.tsx

```diff
@@ -82,18 +84,28 @@ const AreaSelect: FC<Props> = ({
     .reduce((prev, curr) => prev + curr, 0);
 
   return (
-    <>
-      <Box paddingBottom={1}>
+    <Box
+      sx={{
+        display: 'flex',
+        flexDirection: 'column',
+        height: '130vh',
```

Comment:
Why is it hardcoded to be 130vh in height?

Reply:
I hardcoded it to be able to scroll the whole pane-container. Maybe is there a better way to achieve this that I don't know? 🤔

Reply:
I don't fully understand what it's trying to achieve, so I can't offer any recommendation. Maybe we can talk about this @rebecarubio, and figure out a good solution together?